Project-3/README.md

```diff
@@ -8,70 +8,431 @@ For any question involving the use of Protege, please be sure to import:
+  
+  The counterexample can be the cases that involves possible worlds. For example, m is aligned with n in the actural world, and n is aligned with q in a possible world, then it may not be the case that m is aligned with q. 
+  
   (c)  Asymmetric
-  (d)  Functional 
+  
+  Answer: has role in modeling.
+  The definition: A relation between a biological, experimental, or computational artifact and an entity it is used to study, in virtue of its replicating or approximating features of the studied entity.
+  If a relational quality between artifact x and y is asymmetric per se, then it follows that there are no such cases where the relationship between artifact x and entity y is symmetric. But it is possible that an artifact x's relation to entity y may be in some cases symmetric.  
+ 
+
+  (d)  Functional
+  
+  Answer: characteristic of 
+  The definition: a relation between a spefically dependent continaunt (the characteristic) and any other entity (the bearer), in which the characteristic depends on the bearer for its existence. The RO "characteristic of" is functional, which means, if x is characteristic of y, there is only one bearer y for any x. However, for a given characteristic, there can be more than one bearer. 
+     
+  
   (e)  Inverse Functional
-```
+  
+  Answer: has charcteristic 
+  The definition is the inverse of “characteristic of”. This answer relies on what was said in (d), but is its inversion. 
+
 
 3. Model the following natural language expressions using terms from BFO and RO; you are welcome to introduce new terms where needed:  
-```
+
   (a) Sally has an arm Tuesday but does not have an arm Wednesday. 
+  
+   Answer:  "Sally participates in having at least one arm on Tuesday" and "Sally participates in having no arms on Wednesday"
+   
+           Sally is instance_of object
+           Tuesday is instance_of one-dimensional temporal region 
+           Wednesday is an instance of one-dimensional temporal region 
+           Arm is an instance of fiat object part
+           “Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
+           “Sally participates in having no arms on Wednesday” is an instance of occurrent
+           We don't know whether Tuesday precedes Wednesday. 
+   
   (b) Every liver has some cell as part at all times it exists.
+  
+   Answer: Liver has_part_at_all_times Cell
+   
+   Liver is a class in Uberon (UBERON_0002107), which is a subclass of material entity. 
+   We have already had Cell in the Ontology. 
+  
   (c) John was a child, then an adult, then a senior. 
+  
+  Answer:  John participates in childhood precedes John participates in adulthood which precedes John participates in seniorhood
+  
+     John is an instance_of object
+     John's childhood is an instance_of occurrent.
```

Comment:
You need to put these items together to represent the natural language expression.